.github/workflows/release.yaml

```diff
@@ -54,7 +61,13 @@ jobs:
       - uses: ncipollo/release-action@v1
         with:
           allowUpdates: false
-          artifacts: "executables.zip,assets/robotmk_core-1.0.0.mkp"
+          artifacts: >
+            executables.zip,
```

Comment:
The name `executables.zip` seems inconsistent now? Should we not change this?

Reply:
What would you change it to? "all_executables" or so? It still contains the same files as before.

Reply:
Done